benches/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "benches"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.61"
+publish = false
+
+[dependencies]
```

Comment:
The empty `dependencies` section can be removed.

Reply:
Done in #52